VEHICLE/SetVehicleOnGroundProperly.md

```diff
@@ -17,6 +17,8 @@ NativeDB Added Parameter 2: float p1
 ```
 
 ## Parameters
-* **vehicle**: 
+* **vehicle**: The vehicle to set on the ground properly
+* **p1**: Might correlate to the vehicle size that needs to be set on the ground. Values used in scripts: 0f, 1f, 5f, 7f, 7.5f(for bikes apparently), 8f
```

Comment:
Remove this parameter (p1) from the parameters section so there's no mismatch when verifying parameter count with argument declaration count declared at the top of the native.